packages/cli/src/models/initializer/truffle/Truffle.ts

```diff
@@ -72,8 +72,12 @@ const Truffle = {
         .filter((name) => {
           const contract = FileSystem.parseJsonIfExists(`${buildDir}/${name}`);
           if (contract) {
+            const isLibrary = contract.ast ? contract.ast.nodes[1].contractKind === "library" : false;
```

Comment:
I believe this is not generic enough. Image that you have a `Foo.sol` in which you have, some contracts and libraries defined:
```solidity
pragma solidity ^0.5.0;

contract MyContract {}
library MyLibrary {}
library MyOtherLibrary {}
```
In this case, when reading the contract ast, you will find the pragma-related info in the first position and the `MyContract` contract-related info in the second position (i.e., `contract.ast[1]`), and not the info related to the library. You will struggle with the same problem if you have imports before declaring your libraries.
In order to solve this, you can check not only the `contractKind` value but also the `name` to be sure you are picking the right element of the `nodes` array.
For instance:
```ts
const isLibrary = !!contract.ast.nodes.find(node => node.contractKind === 'library' && node.name === contract.contractName);
```
(Please double check that I'm not commiting any mistake with the node's attributes :laughing:)

Reply:
nit: use single quotes instead of double quotes or the linter will complain.